src/map/items.h

```diff
@@ -1,32 +1,34 @@
 ﻿#ifndef _ITEMS_H
 #define _ITEMS_H
 
-// Add items as needed, with 14K items in the database this
-// file would be huge if we put all of them in here.
+// Only add items as utilized in core.  There is no need to maintain a complete list
+// here.
```

Comment:
I'm assuming this list supposed to be the c++ equivalent of enum/item.lua?

Reply:
Yep, but that define style is definitely not the way to do this in the modern age.  Just updating to something a little more current